db/bigtable_eth1.go

```diff
@@ -1609,6 +1609,74 @@ func (bigtable *Bigtable) TransformUncle(block *types.Eth1Block, cache *freecach
 	return bulkData, bulkMetadataUpdates, nil
 }
 
+// TransformWithdrawals accepts an eth1 block and creates bigtable mutations.
+// It transforms the withdrawals contained within a block, extracts the necessary information to create a view and writes that information to bigtable
+// It writes uncles to table data:
+// Row:    <chainID>:W:<reversePaddedNumber>:<withdrawalIndex>
```

Comment:
Consider also reversing the withdrawal index otherwise it will not be a proper descending order

Reply:
done